pages/conference/young-inspirers.vue

```diff
@@ -203,6 +232,128 @@ export default {
                 speech4: require('~/static/img/icons/lang/ZHZH.svg'),
             },
             popupIndex: null,
+            hosts: [
+                {
+                    name: 'DJ',
+                    avatar: require('~/static/img/young-inspirers/host1.png'),
+                },
+                {
+                    name: 'Wey-Han',
+                    avatar: require('~/static/img/young-inspirers/host2.png'),
+                },
+                {
+                    name: '李佳穎',
+                    avatar: require('~/static/img/young-inspirers/host3.png'),
+                },
+                {
+                    name: 'Justin',
+                    avatar: require('~/static/img/young-inspirers/host4.png'),
+                },
+            ],
+            speechInfos: [
+                {
+                    date: '23',
+                    month: 'July',
+                    speeches: [
+                        {
+                            tag: 'speech1',
+                            speechdate: 'July 23',
+                            speechtime: '14:00-14:40 (GMT+8)',
+                            title: 'Learning Python - A Journey',
+                            speaker: [
+                                {
+                                    name: 'Cheung Chun Lok Amos',
+                                    description: [
+                                        'I started my Python journey in Primary 4 when I joined Hour of Code Hong Kong. I was first introduced to programming and Python through the free Hour of Code lectures and workshops. The Hour of Code community provided me with the first resources I had to learn Python and STEM knowledge with, such as book recommendations and learning sites. Through the Internet, books, online documentations, learning sites and especially talks from peers and experts with experience in Python, I started to learn to make small projects and games with packages such as Tkinter and Turtle, which I’d share with others and get feedback from. Slowly, I began to develop a deep interest and passion for learning programming and for coding Python, a passion that I still have today. The more I learn about Python, the more I discover the infinite possibilities and projects I can create with it. Python is a popular language with a lot of potential for usage in machine learning and data science, and I hope to continue learning more and experimenting more on using Python in those fields and packages designed for those purposes, such as Tensorflow and Pandas, in the future.',
```

Comment:
nit: break the long line into multiple strings (concatenate with plus operator) for readability